ppr-ui/tests/unit/TombstoneDischarge.spec.ts

```diff
@@ -58,28 +58,35 @@ describe('Tombstone component', () => {
   }
   const registrationType = mockedSelectSecurityAgreement()
 
-  beforeEach(async () => {
-    // mock the window.location.assign function
-    delete window.location
-    window.location = { assign: jest.fn() } as any
-
+  beforeAll(async () => {
```

Comment:
We don't change the data in the store in these tests, so we can just set it once.